### PR TITLE
Update volk to 1.3.275 & make vulkaninfo use volk for function loading

### PR DESCRIPTION
**Pull request by charles-lunarg**

### CODE REVIEW

File: vulkaninfo/vulkaninfo.cpp [hunk 1]
```diff
@@ -935,21 +934,28 @@ const char *help_message_body =
     "[-o <filename>, --output <filename>]\n"
     "                    Print output to a new file whose name is specified by filename.\n"
     "                    File will be written to the current working directory.\n"
-    "[--text]            Produce a text version of " APP_SHORT_NAME " output to stdout. This is\n"
+    "[--text]            Produce a text version of " APP_SHORT_NAME
+    " output to stdout. This is\n"
```
Comment (lunarpapillo) [inline vulkaninfo/vulkaninfo.cpp:938]: are these (and below) `clang-format` changes?  They seem unrelated to the intended changes.

File: vulkaninfo/vulkaninfo.h [hunk 2]
```diff
@@ -1196,7 +973,7 @@ static VkSurfaceKHR AppCreateGgpSurface(AppInstance &inst) {
     createInfo.streamDescriptor = 1;
 
     VkSurfaceKHR surface;
-    VkResult err = inst.dll.fp_vkCreateStreamDescriptorSurfaceGGP(inst.instance, &createInfo, NULL, &surface);
+    VkResult err = vkCreateStreamDescriptorSurfaceGGP(inst.instance, &createInfo, NULL, &surface);
```
Comment (lunarpapillo) [inline vulkaninfo/vulkaninfo.h:976]: This one seems unusual; I'm curious to know why we're supporting a `GGP` extension (is that Google?).

**Review by richard-lunarg — APPROVED** [review]
Ship it.

**Review by lunarpapillo — APPROVED** [review]
Code looks good; I have a couple of noob questions, but those don't need to stop this merge.

File: vulkaninfo/vulkaninfo.h (hunk 2 above)
Comment (charles-lunarg) [reply vulkaninfo/vulkaninfo.h:976]: Because it was added at some point. GGP means google games platform, AKA Stadia.

File: vulkaninfo/vulkaninfo.cpp (hunk 1 above)
Comment (charles-lunarg) [reply vulkaninfo/vulkaninfo.cpp:938]: yes, this one sneaked in under the radar.

Comment (charles-lunarg) [reply vulkaninfo/vulkaninfo.cpp]: Looked through for accidental formatting changes and reverted them - I suspect my local clang-format is "different" and don't want to think about formatting changes today.
